.editorconfig

```diff
@@ -5,7 +5,7 @@ root = true
 
 [*]
```

Comment:
I think this file is not needed.
The code style will be found by your editor ( at least vscode does) from your prettier/linter config files